src/Chapter1.hs

```diff
@@ -615,7 +624,13 @@ Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
 
-sumLast2 n = error "sumLast2: Not implemented!"
+
+sumLast2 :: Integral a => a -> a
+sumLast2 n = 
+   let 
+      last2 = n `mod` 100
+   in
+      (last2 `div` 10) + (last2 `mod` 10)
```

Comment:
That is a wonderful solution! 👏🏼  You correctly noticed that it is the `div` and `mod`, cool 😎 

One hint to make your solution even shorter: you can see that you use both:

```haskell
last2 `mod` 10
last2 `div` 10
```
The standard library has the [`divMod`](https://hackage.haskell.org/package/base-4.14.0.0/docs/Prelude.html#v:divMod) function, that actually combines inside both `div` and `mod`. And this is exactly what you use!.

So you could write it this way:

```haskell
(a, b) = last2 `divMod` 10
```

You can see how we could pattern match on the pair 🙂

Also, you need to use `abs` in here to cover negative cases 👌🏼